AgentBlob/GenerateAgentBlob.ps1

```diff
@@ -0,0 +1,79 @@
+﻿
```

Comment:
Add `$ErrorActionPreference = "Stop"` at the beginning of the script.

This will stop the script executing in case an exception is thrown.

Reply:
will do